src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -155,6 +188,58 @@ private IEnumerable<DependencyDetail> FilterToolsetDependencies(IEnumerable<Depe
             return dependencies;
         }
 
+        private void LogBasicNodeDetails(DependencyGraphNode node, string indent)
+        {
+            Console.WriteLine($"{indent}- Repo:     {node.RepoUri}");
+            Console.WriteLine($"{indent}  Commit:   {node.Commit}");
+            GitDiff diffFrom = node.DiffFrom;
+            // Log the node diff if it was desired.
+            if (diffFrom != null)
```

Comment:
I can't figure out what this code is trying to do... some comments about what the string sort of looks like in general and what it's trying to figure out would make it a bit easier.

Reply:
Sure thing.